File: 프로그래머스/lv0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
def solution(my_str, n):
    
    my_str = list(my_str)
    
    result = []
    temp1 = []
      
    for i in range(0, len(my_str)):
        temp1.append(my_str[i])
        
        
        if(len(temp1) % n == 0):
            result.append(str(''.join(map(str, temp1))))
            temp1 = []
        
        elif(i+1 == len(my_str)):
            result.append(str(''.join(map(str, temp1))))
            
    return result

File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.js
function solution(my_str, n) {
    const result = [];
    let temp1 = [];
    
    for (let i = 0; i < my_str.length; i++) {
        temp1.push(my_str[i]);
        
        if (temp1.length % n === 0 || i + 1 === my_str.length) {
            result.push(temp1.join(''));
            temp1 = [];
        }
    }
    
    return result;
}
